module/programs/helper.js

```diff
@@ -78,14 +78,15 @@ module.exports = class ProgramsHelper {
     return new Promise(async (resolve, reject) => {
 
       try {
+
+        if(!data.hasOwnProperty('endDate')){
```

Comment:
@ankitpws Let's make it mandatory, no need to check this here, they must give in API